### PR TITLE
Implemented QE Review feedback for Getting started with the rhoas CLI for Red Hat OpenShift Streams for Apache Kafka

### PR DESCRIPTION
**Pull request by AedinC**
Amended admonition block in Creating a Kafka topic step, removed 4 spaces from json code snippet, and clarified service-account and clientID correlation in Creating a service account step.

### CODE REVIEW

File: docs/kafka/rhoas-cli-getting-started-kafka/README.adoc [hunk 1]
```diff
@@ -180,7 +180,7 @@ You use these credentials and the bootstrap server URL to connect your applicati
 . Set the permissions for your service account to access the Kafka instance resources.
 +
 --
-This example allows applications to produce and consume messages from topics starting with a specified prefix that belong to any consumer group.
+This example allows applications to produce and consume messages from topics starting with a specified prefix that belong to any consumer group. The service account information you input here is the client ID credentials from the previous step.
```
Comment (wtrocki) [inline docs/kafka/rhoas-cli-getting-started-kafka/README.adoc:183]: credentials would be typically associated to username and password. Skipping that word will be better.
```suggestion
This example allows applications to produce and consume messages from topics starting with a specified prefix that belong to any consumer group. The service account information you input here is the clientID from the previous step.
```

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
Amazing change! Thank you for spotting this issues

File: docs/kafka/rhoas-cli-getting-started-kafka/README.adoc (hunk 1 above)
Comment (AedinC) [reply docs/kafka/rhoas-cli-getting-started-kafka/README.adoc:183]: Thanks Wojciech!

**Comment by AedinC** [comment]
> Amazing change! Thank you for spotting this issues

All thanks go to Martin Murphy in QE!

File: docs/kafka/rhoas-cli-getting-started-kafka/README.adoc [hunk 2]
```diff
@@ -180,7 +180,7 @@ You use these credentials and the bootstrap server URL to connect your applicati
 . Set the permissions for your service account to access the Kafka instance resources.
 +
 --
-This example allows applications to produce and consume messages from topics starting with a specified prefix that belong to any consumer group.
+This example allows applications to produce and consume messages from topics starting with a specified prefix that belong to any consumer group. The service account information you input here is the clientID from the previous step.
```
Comment (bhardesty) [inline docs/kafka/rhoas-cli-getting-started-kafka/README.adoc:183]:
```suggestion
This example allows applications to produce and consume messages from topics starting with a specified prefix that belong to any consumer group. The service account information you enter here is the `clientID` from the previous step.
```

**Review by bhardesty — APPROVED** [review]
Looks great, thanks @AedinC! Just had one really minor suggestion.

**Comment by AedinC** [comment]
> Looks great, thanks @AedinC! Just had one really minor suggestion.

Thanks Ben!

**Pull request merged**